File: sdks/thunkmetrc-cs/ThunkMetrc.Wrapper/Models/UpdateSplitRequest.cs
using System;
using System.Collections.Generic;

namespace ThunkMetrc.Wrapper.Models
{
    public class UpdateSplitRequest
    {
        [global::System.Text.Json.Serialization.JsonPropertyName("PlantCount")]
        public int? PlantCount { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("SourcePlantLabel")]
        public string? SourcePlantLabel { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("SplitDate")]
        public string? SplitDate { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("StrainLabel")]
        public string? StrainLabel { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("TagLabel")]
        public string? TagLabel { get; set; }
    }
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/UpdateSplitRequest.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class UpdateSplitRequest(
    @JsonProperty("PlantCount")
    val plantCount: Int? = null,
    @JsonProperty("SourcePlantLabel")
    val sourcePlantLabel: String? = null,
    @JsonProperty("SplitDate")
    val splitDate: String? = null,
    @JsonProperty("StrainLabel")
    val strainLabel: String? = null,
    @JsonProperty("TagLabel")
    val tagLabel: String? = null
)
